lib/iex/lib/iex/server.ex

```diff
@@ -274,13 +279,18 @@ defmodule IEx.Server do
     {_, _, env, scope} = :elixir.eval('import IEx.Helpers', [], env)
 
     binding = Keyword.get(opts, :binding, [])
+    evaluator_state  = %IEx.Evaluator.State{binding: binding, scope: scope, env: env}
+
+    evaluator_state =
+      case opts[:dot_iex_path] do
+        ""   -> evaluator_state
+        path -> IEx.Evaluator.load_dot_iex(evaluator_state, path)
+      end
+
     prefix  = Keyword.get(opts, :prefix, "iex")
```

Comment:
You should either remove the whitespace after `prefix` or indent the line below accordingly. :)